install.json

```diff
@@ -1,54 +1,101 @@
 {
     "resources": {
-        "body": [
+        "head": [
+            {
+                "type": "style",
+                "src": "progress-bar.css"
+            },
             {
                 "type": "script",
-                "src": "script.js"
+                "src": "progress-bar.js"
+            }
+        ]
+    },
+    "preview": {
```

Comment:
This preview handler allows the app to update when options are changed in the Eager install page without reloading the site.